pybop/costs/_likelihoods.py

```diff
@@ -141,7 +141,7 @@ def _add_single_sigma(self, index, value):
                 Parameter(
                     f"Sigma for output {index+1}",
                     initial_value=value,
-                    prior=Uniform(0.5 * value, 1.5 * value),
+                    prior=Uniform(1e-3 * value, 1e3 * value),
```

Comment:
This is extended to ensure the logpdf of the uniform prior is finite across the search space.